### PR TITLE
[OUI DOCS] Updated the content in the section Datagrid and schema

### LINKED ISSUE #228: (OUI Docs) Data grid schemas and popovers
Section: Disabling expansion popovers 
Table column "custom" 
- [ ] Change "Star Wars" to "Oranges"
- [ ] Change "Start Trek" to "Apples"
- [ ] Change popover content when clicking on "Star Wars" - "Star Wars is the best" to "Oranges", alt text "Random star wars image" to "Random orange image" and use url https://unsplash.com/600x600/?oranges
- [ ] Change popover content when clicking on "Star Wars" - "Star treak is the best" to "Apples", alt text "Random star wars image" to "Random apple image" and use url https://unsplash.com/600x600/?apples

### PR DESCRIPTION
**Pull request by AbhishekReddy1127**
Signed-off-by: AbhishekReddy1127 <nallamsa@amazon.com>

### Description
Updated the content for the Section: Disabling expansion popovers and Table column "custom"

Screenshot 2023-02-08 at 12 51 07 PM

 
### Issues Resolved
#228 
 
### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] All tests pass
  - [x] `yarn lint`
  - [x] `yarn test-unit`
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: src-docs/src/views/datagrid/datagrid_example.js [hunk 1]
```diff
@@ -120,29 +120,29 @@ const gridSnippet = `
...
+    // This schema 'franchise' essentially acts like a boolean, looking for Oranges or Apples in a column.
     schemaDetectors={[
       {
         type: 'franchise',
         // Try to detect if column data is this schema. A value of 1 is the highest possible. A (mean_average - standard_deviation) of .5 will be good enough for the autodetector to assign.
         detector(value) {
-          return value.toLowerCase() === 'star wars' ||
-            value.toLowerCase() === 'star trek'
+          return value.toLowerCase() === 'Oranges' ||
+            value.toLowerCase() === 'Apples'
```
Comment (BSFishy) [inline src-docs/src/views/datagrid/datagrid_example.js:130]: It looks like here `Oranges` and `Apples` should be lowercase

File: src-docs/src/views/datagrid/datagrid_example.js [hunk 2]
```diff
@@ -120,29 +120,29 @@ const gridSnippet = `
...
             ? 1
             : 0;
         },
         // How we should sort data matching this schema. Again, a value of 1 is the highest value.
         comparator(a, b, direction) {
-          const aValue = a.toLowerCase() === 'star wars';
-          const bValue = b.toLowerCase() === 'star wars';
+          const aValue = a.toLowerCase() === 'Oranges';
+          const bValue = b.toLowerCase() === 'Oranges';
```
Comment (BSFishy) [inline src-docs/src/views/datagrid/datagrid_example.js:137]: See previous comment

File: src-docs/src/views/datagrid/datagrid_example.js [hunk 3]
```diff
@@ -120,29 +120,29 @@ const gridSnippet = `
...
           if (aValue < bValue) return direction === 'asc' ? 1 : -1;
           if (aValue > bValue) return direction === 'asc' ? -1 : 1;
           return 0;
         },
         // Text for what the ASC sort does.
-        sortTextAsc: 'Star Wars-Star Trek',
+        sortTextAsc: 'No data available-Apples',
```
Comment (BSFishy) [inline src-docs/src/views/datagrid/datagrid_example.js:143]: Should this be `Oranges-Apples`?

File: src-docs/src/views/datagrid/schema.js [hunk 4]
```diff
@@ -87,26 +87,26 @@ const Franchise = (props) => {
...
+          url="https://source.unsplash.com/600x600/?oranges"
         />
       ) : (
         <OuiImage
           allowFullScreen
           size="m"
           hasShadow
-          caption="Random star trek image"
+          caption="Random Apples image"
           alt="Random trek image"
```
Comment (BSFishy) [inline src-docs/src/views/datagrid/schema.js:108]: Should this alt text be changed as well?

File: src-docs/src/views/datagrid/schema.js [hunk 5]
```diff
@@ -87,26 +87,26 @@ const Franchise = (props) => {
   return (
     <div>
       <OuiTitle size="s">
-        <h3>{props.name} is the best!</h3>
+        <h3>{props.name} Oranges </h3>
```
Comment (BSFishy) [inline src-docs/src/views/datagrid/schema.js:90]: Should the ` Oranges ` here be entirely removed?

File: src-docs/src/views/datagrid/schema.js [hunk 6]
```diff
@@ -193,20 +193,20 @@ const DataGridSchema = () => {
           type: 'favoriteFranchise',
           textTransform: 'capitalize',
           detector(value) {
-            return value.toLowerCase() === 'star wars' ||
-              value.toLowerCase() === 'star trek'
+            return value.toLowerCase() === 'Oranges' ||
+              value.toLowerCase() === 'Apples'
```
Comment (BSFishy) [inline src-docs/src/views/datagrid/schema.js:197]: Same capitalization comment as above

File: src-docs/src/views/datagrid/schema.js [hunk 7]
```diff
@@ -193,20 +193,20 @@ const DataGridSchema = () => {
...
               ? 1
               : 0;
           },
           comparator(a, b, direction) {
-            const aValue = a.toLowerCase() === 'star wars';
-            const bValue = b.toLowerCase() === 'star wars';
+            const aValue = a.toLowerCase() === 'Oranges';
+            const bValue = b.toLowerCase() === 'Oranges';
```
Comment (BSFishy) [inline src-docs/src/views/datagrid/schema.js:203]: Same capitalization comment as above

**Review by BSFishy — APPROVED** [review]

**Review by seanneumann — DISMISSED** [review]
👍

**Review by BSFishy — DISMISSED** [review]

**Review by BSFishy — APPROVED** [review]

**Review by seanneumann — APPROVED** [review]
👍

**Pull request merged**